api/client/webclient/webclient_test.go

```diff
@@ -289,3 +290,12 @@ func TestExtract(t *testing.T) {
 		})
 	}
 }
+
+func TestNewWebClientRespectHTTPProxy(t *testing.T) {
+	os.Setenv("HTTPS_PROXY", "localhost:9999")
+	defer os.Unsetenv("HTTPS_PROXY")
+	client := newWebClient(false, nil)
```

Comment:
Comment ambiguous arguments?

```suggestion
	client := newWebClient(false /* insecure */, nil /* pool */)
```